features/reporting.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Covering letter
+  As Pension Wise
+  We want to store customer details for reporting
+  So we can use the result to provide a better service
+
+  Scenario: Details of the appointment are stored for reporting
+    Given a customer has had a Pension Wise appointment
```

Comment:
I agree, but feel this should happen in a refactor story/PR so as not to make this commit just keep growing.

I used the existing step, and while changing the wording in it is a small change, I feel this is a gateway drug to refactoring hell without achieving the sprint goals first.